### PR TITLE
Update push.yml

### PR DESCRIPTION
**Pull request by cschuchardt88**
### ONLY MERGE IF NEEDED FOR FIX https://github.com/neo-project/neo-express/pull/358#issuecomment-1823783942

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -36,4 +36,4 @@ jobs:
         path: ./out
 
     - name: Publish to Azure Artifacts
-      run: dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${MYGET_TOKEN} 
+      run: dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
```
Comment (Liaojinghui) [inline .github/workflows/push.yml:39]: Do you still need token, or its already set?

**Review by Liaojinghui — APPROVED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (cschuchardt88) [reply .github/workflows/push.yml:39]: from the error is says it isnt set

Comment (cschuchardt88) [reply .github/workflows/push.yml:39]: can you check

Comment (Liaojinghui) [reply .github/workflows/push.yml:39]: @shargon can you help?

**Comment by cschuchardt88** [comment]
```
Run dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${MYGET_TOKEN}
Required argument missing for option: '--api-key'.

Specify --help for a list of available options and commands.
error: Missing value for option 'api-key'
```

**Comment by Liaojinghui** [comment]
> ```
> Run dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${MYGET_TOKEN}
> Required argument missing for option: '--api-key'.
> 
> Specify --help for a list of available options and commands.
> error: Missing value for option 'api-key'
> ```

Wait a minute, did you run it on your own repo? or you run it here?

**Comment by cschuchardt88** [comment]
no i didnt https://github.com/neo-project/neo-express/actions/runs/6965238723

**Comment by cschuchardt88** [comment]
looks like it working now.

**Comment by Liaojinghui** [comment]
@cschuchardt88  

    - name: Upload Build Log
      if: always()
      uses: actions/upload-artifact@v3
      with:
        name: buildlog
        path: ./msbuild.binlog

    - name: Upload NeoExpress
      uses: actions/upload-artifact@v3
      with:
        name: packages
        path: ./out/*.nupkg
        
        This creates a 90MB file to github storage on every run, can you figure out a way to avoid this? maybe no upload and download process but merge pack and publish jobs together?

**Comment by cschuchardt88** [comment]
@Liaojinghui 
image

i didnt see symbol get uploaded? Can you check?

myget says for symbols
https://docs.myget.org/docs/reference/symbols
```
If .nupkg and .snupkg packages are in the same folder, you only need to us push command for nupkg (we will detect the .snupkg and push it for you):
```

**Comment by Liaojinghui** [comment]
It is even enabled? i did not see its configed in neoxp

**Comment by Liaojinghui** [comment]
By the way,lets try to work on development branch

**Comment by cschuchardt88** [comment]
@Liaojinghui its in `Directory.Build.props`
```xml
<SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

**Comment by cschuchardt88** [comment]
@Liaojinghui now if we combine pack and publish, then it will publish on PRs too? is that ok?

**Comment by Liaojinghui** [comment]
No, cant way not run pack on pr?

**Comment by cschuchardt88** [comment]
Look at #371 i think its ok?

**Comment by Liaojinghui** [comment]
> @Liaojinghui its in `Directory.Build.props`
> 
> ```
> <SymbolPackageFormat>snupkg</SymbolPackageFormat>
> ```

    <IncludeSymbols>true</IncludeSymbols>
    <SymbolPackageFormat>snupkg</SymbolPackageFormat>

**Pull request merged**